explainability-core/src/main/java/org/kie/trustyai/explainability/model/PerturbationContext.java

```diff
@@ -27,21 +27,34 @@
  */
 public class PerturbationContext {
 
+    private static final double DEFAULT_STANDARD_DEVIATION = 0.01;
     private final Optional<Long> seed;
 
     private final Random random;
 
     private final int noOfPerturbations;
 
+    private final double standardDeviation;
+
     public PerturbationContext(Random random, int noOfPerturbations) {
+        this(random, noOfPerturbations, DEFAULT_STANDARD_DEVIATION); // preserving old behavior
```

Comment:
Do we want to preserve the old behaviour? If we are changing this behaviour because suboptiman/broken I would probably change also this. I don't think it is an issue to change this

Reply:
I think this can be safely changed, I just would like to get evidence from real benchmarks to choose the new value. I've adjusted the LIME retry mechanism to multiply this by 10 automatically on each retry, so it'll be 0.01 on the first try, 1 on the second, 10 on the third, etc.
tldr; I'd keep this until we get a better default from experiments with real data.